### PR TITLE
Add S02E01 video #262

### LINKED ISSUE #262: Please add content to OpenHW website
We would like the following news item hosted on the OpenHW site. As usual, this needs to be on a page on OpenHW website. https://www.openhwgroup.org/resources/openhwtv/
Note there will be a youku.com link to follow but let's please not wait for this before publishing.
I will send that as an update later. Thanks.
Here is the content:

**Date** 
Thursday, 28 January, 2021

**Title:** 
OpenHW TV Episode 1 of Season 2 now available to view

**Body:**
2021. New year, new episode of OpenHW TV. In the first episode of Season Two, we feature an interview between embedded.com editor-in-chief Nitin Dahad and OpenHW Group President and CEO Rick O'Connor. They discuss the latest milestones of the OpenHW Group, the roadmap for the organization, as well as the key issues that are driving the growth of open-source processor design and development. Plus there's a brief bonus round-up of season one showing the interesting range of guidance and discussion that was presented last year.
You can access the entire recording here: https://youtu.be/LZ8N3uLD4wY

OpenHWTV-S2E1-visual

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #262 

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

**Comment by flora8984461** [comment]
FYI: We don't need to wait for Youku, they will post later.

File: content/resources/openhwtv/s02e01.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "OpenHW TV S02 E01"
+date: 2021-01-28
+season: 2
+episode_title: embedded.com interview and Season One round-up
+---
+
+{{< youtube "LZ8N3uLD4wY" >}}
+
+2021.New year, new episode of OpenHW TV. In the first episode of Season Two, we feature an interview between embedded.com editor-in-chief Nitin Dahad and OpenHW Group President and CEO Rick O'Connor. They discuss the latest milestones of the OpenHW Group, the roadmap for the organization, as well as the key issues that are driving the growth of open-source processor design and development. Plus there's a brief bonus round-up of season one showing the interesting range of guidance and discussion that was presented last year.
```
Comment (autumnfound) [inline content/resources/openhwtv/s02e01.md:10]: Small nitpick, add a space between `2021.` and `New` at the start of this line

Comment (flora8984461) [reply content/resources/openhwtv/s02e01.md]: thanks for pointing, actually got one more updates including this one from NaylDSouza

**Review by autumnfound — APPROVED** [review]
Not sure why we need the slash, but it looks good on output, so I'll merge

**Comment by NaylDSouza** [comment]
@flora8984461 *Not urgent, but we now have the Youku link and that closes the update and makes this complete*

View on youku.com 

Link: https://v.youku.com/v_show/id_XNTA5MDQwMDE1Ng==.html

**Pull request merged**